Subset_Sum/Subset_Sum.cpp

```diff
@@ -0,0 +1,83 @@
+/* Subset Sum Problem: Given an array of elements and a sum,
+is it possible to form the given sum using some or all of the
+elements of array i.e using any subset of given array.
+Example: 1, 7, 3, 5, 6 and sum = 14, it is possible to
+obtain the sum 14 by taking the elements 1, 7 and 6 */
+
+#include <bits/stdc++.h>
+using namespace std;
+typedef long long int ll;
+
+// Function to compute whether a subset of array exists
+// having the given sum
+bool SubsetSum(ll arr[], ll size, ll sum) 
```

Comment:
function names in c++ are always camel case. The above style of naming is done in C#

Reply:
done